app/models/call_result.rb

```diff
@@ -22,6 +22,7 @@ class CallResult < ApplicationRecord
   enum remove_reason: {
     already_visited: "already_visited",
     not_responding: "not_responding",
+    refused_to_come_back: "refused_to_come_back",
     invalid_phone_number: "invalid_phone_number",
```

Comment:
We'll need to update a couple more places:
- In `Appointment#update_patient_status`, I think if the patient has refused to come back, they should be marked as `migrated`.
- Add it to `Patient#could_not_contact_reasons`
- In the dashboard overdue list `views/appointment/index.html`, we should add this reason in the dropdown to mark overdue results

I looked for `public_hospital_transfer` across the codebase, should check if there's any other touchpoints.

Reply:
Lets also regenerate the swagger docs (by running `rake docs`) to update the enum there.

Reply:
Thanks, @prabhanshuguptagit. I have to check with someone if we should be marking these patients as `migrated`.

Reply:
@prabhanshuguptagit `refused_to_come_back` doesn't necessarily mean that they have `migrated` right?

Reply:
Also, `Patient#could_not_contact_reason` seems independent from `remove_reason`, in that case, do we need to add it here?

Reply:
@danySam Yeah actually, think it makes sense to not add it to`Patient#could_not_contact_reason`. Seems like it's also dead code so unsure what context it was supposed to be used in.